package.json

```diff
@@ -41,6 +41,7 @@
     "eslint": "8.51.0",
     "eslint-config-next": "13.5.4",
     "firebase": "^10.4.0",
+    "immer": "^10.0.3",
```

Comment:
Personally I don't convenient with how immer works. I think it adding unnecessary complexity in the code.

I don't think it is good to introduce this in the current stage.

Reply:
How about we remove this one?

Reply:
immer is usually used with zustand to make the code more readable

sure, let me try to refactor them

Reply:
I already used zustand in many projects also, but without immer.

And I can say, it still works like charm.

So based on my experiences, it's just not needed in my workflow.

Of course, this is just personal preference.